src/api/java/appeng/api/networking/energy/IEnergyWatcherNode.java

```diff
@@ -27,7 +27,7 @@
 
 import appeng.api.networking.IGridNodeService;
 
-public interface IEnergyWatcherHost extends IGridNodeService {
+public interface IEnergyWatcherNode extends IGridNodeService {
```

Comment:
`Host` is actually the more fitting name here. The actual watchers are implemented by use and act as the observer part. No idea what pattern here really applies.

Reply:
Renamed because of `ICraftingWatcherNode` (renamed during the grid refactor). The host referred to `IGridHost` imo, although I guess that's up to interpretation.